packages/actions/entity-report/src/main.js

```diff
@@ -0,0 +1,72 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import {appFactory, externalEvents, selection} from 'tocco-app-extensions'
+import {reducer as reducerUtil} from 'tocco-util'
+
+import SelectionContainer from './components/Selection/SelectionContainer'
+import reducers, {sagas} from './modules/reducers'
+
+const packageName = 'entity-report'
+
+const EXTERNAL_EVENTS = ['onSuccess', 'onError']
+
+const initApp = (id, input, events, publicPath) => {
+  const content = <SelectionContainer />
+  const store = appFactory.createStore(reducers, sagas, input, packageName)
+  externalEvents.addToStore(store, events)
+
+  return appFactory.createApp(packageName, content, store, {
+    input,
+    events,
+    actions: [],
+    publicPath,
+    textResourceModules: ['actions', 'component', 'common', packageName]
+  })
+}
+
+;(() => {
+  if (__PACKAGE_NAME__ === packageName) {
+    appFactory.registerAppInRegistry(packageName, initApp)
+
+    if (__DEV__) {
+      const input = require('./dev/input.json')
+
+      if (!__NO_MOCK__) {
+        const fetchMock = require('fetch-mock').default
+        fetchMock.config.overwriteRoutes = false
+        const setupFetchMocks = require('./dev/fetchMocks').default
+        setupFetchMocks(packageName, fetchMock)
+        fetchMock.spy()
+      }
+
+      const app = initApp(packageName, input)
+
+      if (module.hot) {
+        module.hot.accept('./modules/reducers', () => {
+          const reducers = require('./modules/reducers').default
+          reducerUtil.hotReloadReducers(app.store, reducers)
+        })
+      }
+
+      appFactory.renderApp(app.component)
+    }
+  }
+})()
+
+const EntityReportApp = props => {
+  const {component} = appFactory.useApp({initApp, props, packageName, externalEvents: EXTERNAL_EVENTS})
+  return component
+}
+
+EntityReportApp.propTypes = {
+  selection: selection.propType.isRequired,
+  actionProperties: PropTypes.shape({
+    path_to_reports: PropTypes.string.isRequired
```

Comment:
Could we rename this to `pathToReports`? Or why do we have this snake case naming here?